render/mapping.go

```diff
@@ -121,7 +121,12 @@ func MapContainerIdentity(m RenderableNode, _ report.Networks) RenderableNodes {
 		rank  = m.Metadata[docker.ImageID]
 	)
 
-	return RenderableNodes{id: NewRenderableNodeWith(id, major, minor, rank, m)}
+	node := NewRenderableNodeWith(id, major, minor, rank, m)
+	if imageID, ok := m.Metadata[docker.ImageID]; ok {
+		hostID, _, _ := report.ParseContainerNodeID(m.ID)
+		node.Origins = node.Origins.Add(report.MakeContainerNodeID(hostID, imageID))
```

Comment:
1. A renderable node's origins are defined to be the topology nodes that have been merged to produce it. You're not merging the container image node into this one, so it's not appropriate to use origins.
2. Minor point compared to the above, but: you're overloading MakeContainerNodeID, i.e. "make a node ID for the container topology", and using it to mean "make a node ID for the container image topology".

If you want to display the image ID in the detail pane, and you know the `docker.ImageID` is present in the metadata for that RenderableNode, then you can put this logic in the thing which generates the detail pane, and leave origins untouched, I think?